README.md

```diff
@@ -73,6 +73,39 @@ Guide](https://solr.apache.org/guide/solr/latest/getting-started/solr-tutorial.h
 - Slack: Solr Community Channel.  Sign up at https://s.apache.org/solr-slack
 - IRC: `#solr` on [libera.chat](https://web.libera.chat/?channels=#solr)
 
+## Developer Documentation
+
+Solr uses Gradle for its build system. Here some useful hints to build and run Solr locally:
```

Comment:
```suggestion
Solr uses Gradle for its build system. Here are some useful hints to build and run Solr locally:
```

Reply:
thanks, I've modified my latest commit with this correction.